com.unity.formats.fbx/Editor/Sources/Recorders/FbxRecorder/FbxRecorderSettings.cs

```diff
@@ -22,6 +22,167 @@ public bool ExportGeometry
             }
         }
 
+        [SerializeField]
+        private string m_animSourceBindingId;
+        [SerializeField]
+        private string m_animDestBindingId;
+
+        public Transform TransferAnimationSource
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(m_animSourceBindingId))
+                    return null;
+
+                return GetBinding(m_animSourceBindingId);
+            }
+
+            set
+            {
+                if (!TransferAnimationSourceIsValid(value))
+                {
+                    return;
+                }
+                if (string.IsNullOrEmpty(m_animSourceBindingId))
+                    m_animSourceBindingId = GenerateBindingId();
+
+                SetBinding(m_animSourceBindingId, value);
+            }
+        }
+
+        public Transform TransferAnimationDest
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(m_animDestBindingId))
+                    return null;
+
+                return GetBinding(m_animDestBindingId);
+            }
+
+            set
+            {
+                if (!TransferAnimationDestIsValid(value))
+                {
+                    return;
+                }
+                if (string.IsNullOrEmpty(m_animDestBindingId))
+                    m_animDestBindingId = GenerateBindingId();
+
+                SetBinding(m_animDestBindingId, value);
+            }
+        }
+
+        static Transform GetBinding(string id)
```

Comment:
Please comment what this is for (it's for domain reload, right?)

Reply:
You didn't say why this function exists -- why do we need binding IDs at all.

Reply:
Oh wait you do a bit later, in GenerateBindingIDs. Maybe move that function to be the first of the bunch?